polaris.shopify.com/pages/api/v0/tokens/[tokens].tsx

```diff
@@ -1,64 +1,30 @@
+import {
+  TokenGroup,
+  ColorScheme,
+  createVar,
+  tokens,
+} from '@shopify/polaris-tokens';
 import type {NextApiRequest, NextApiResponse} from 'next';
-import fetch from 'node-fetch';
-
-interface TokenGroup {
-  [token: string]: string;
-}
-
-/**
- * Colors token groups
- */
-export const colorTokenGroupKeys = ['color.light', 'color.dark'] as const;
-
-type ColorTokenGroupKey = typeof colorTokenGroupKeys[number];
 
 /**
  * Color scheme independent token groups
  */
 export const staticTokenGroupKeys = [
   'depth',
-  'legacy-tokens',
+  'legacyTokens',
   'motion',
   'shape',
   'spacing',
   'typography',
-  'z-index',
+  'zIndex',
 ] as const;
```

Comment:
Perhaps we should derive this data structure now that we have direct access to the tokens e.g.
```ts
import {tokens, Tokens} from '@shopify/polaris-tokens';

type StaticTokenGroupKey = Exclude<keyof Tokens, 'colorSchemes'>;

const staticTokenGroupKeys = Object.keys(tokens).filter(
  (token) => token !== 'colorSchemes',
) as StaticTokenGroupKey[];
```